mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -97,9 +101,13 @@ def binary_cross_entropy(pred,
     """
     # The default value of ignore_index is the same as F.cross_entropy
     ignore_index = -100 if ignore_index is None else ignore_index
-    if pred.dim() != label.dim():
-        label, weight = _expand_onehot_labels(label, weight, pred.size(-1),
-                                              ignore_index)
+    if expand_one_hot:
```

Comment:
Can you give an example of `pred` and `label` for `expand_one_hot=False` in Mask2Former?
What is the difference with  `use_mask=True`?

Reply:
The value of `label` we get from `_expand_onehot_labels` is 0 or 1, but in mask2former label passed to here is the output of `point_sample`, which is float, may be values like `0.6487, 0.6335, 0.5609`.

Reply:
What if use_mask=True?